src/semant/translate.sml

```diff
@@ -150,4 +151,7 @@ structure Translate : TRANSLATE = struct
                                  T.LABEL(join)]),
                                T.CONST(0))))
     end
+
+  fun sequence(l: exp list) =
+    Ex(Tree.ESEQ(l))
```

Comment:
Isn't that what I wrote? I'm confused by the type error in getting.

---
Nathan Lilienthal
@nixpulvis (everywhere)

> On Mar 23, 2016, at 9:13 PM, Gary Soeller <notifications@github.com> wrote:
> 
> In src/semant/translate.sml:
> 
> > @@ -150,4 +151,7 @@ structure Translate : TRANSLATE = struct
> >                                   T.LABEL(join)]),
> >                                 T.CONST(0))))
> >      end
> > +
> > +  fun sequence(l: exp list) =
> > +    Ex(Tree.ESEQ(l))
> Yes it does... In that case, this should probably just be a recursive function which individually adds each exp
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>